fahmunge/cli.py

```diff
@@ -105,7 +105,7 @@ def main():
 
     # Main processing loop
     iteration = 0
-    while((args.maximum_iterations==None) or (iteration < args.maximum_iterations)):
+    while((args.maximum_iterations == None) or (iteration < args.maximum_iterations)):
```

Comment:
In general, it's better to use `is None` rather than `== None` (although in practice it looks like it doesn't matter that much) ref: http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html